nextflow/modules/autometa_core/contig_binning/binning_workflow.nf

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+include { BINNING } from './process/binning.nf'
+include { UNCLUSTERED_RECRUITMENT } from './process/unclustered_recruitment.nf'
+include { BINNING_SUMMARY } from './process/binning_summary.nf'
+
+
+workflow BIN_CONTIGS {
+
+    take:
+    metagenome
+    kmers_embedded
+    kmers_normalized
+    kmer_coverage_results
+    gc_content
+    markers_results
+    taxon_assignment_results
+    binning_column
+    
+
+    main: 
+        BINNING(kmers_embedded, kmer_coverage_results, gc_content, markers_results, taxon_assignment_results)
+        UNCLUSTERED_RECRUITMENT(kmers_normalized, kmer_coverage_results, BINNING.out.binning, markers_results, taxon_assignment_results)
+        BINNING_SUMMARY(BINNING.out.main, markers_results, metagenome, binning_column)
+  
+    emit:
+        binning = BINNING.out.binning
+        binning_main = BINNING.out.main
+        recruitment = UNCLUSTERED_RECRUITMENT.out.binning
+        recruitment_main = UNCLUSTERED_RECRUITMENT.out.main
+        all_binning_results = BINNING.out.binning | mix(UNCLUSTERED_RECRUITMENT.out) | collect
```

Comment:
It's what was in dev https://github.com/KwanLab/Autometa/blob/50f7a60c329acfa9e705b4cf343dfdcc21ade0ec/nextflow/autometa.nf#L44

Not sure if it was what you intended because It just collects everything for output, but these emit channels aren't ever used because they are at the end of the pipeline